packages/e2e/test/wallet_epoch_3/SharedWallet/delegation.test.ts

```diff
@@ -204,6 +204,16 @@ describe('SharedWallet/delegation', () => {
     await waitForTx(aliceMultiSigWallet, tx.id);
     const tx1ConfirmedState = await getWalletStateSnapshot(aliceMultiSigWallet);
 
+    // Check Registration and StakeDelegation certificate from ChainHistoryProvider
+    let gotTx = toSerializableObject(
+      (await firstValueFrom(aliceMultiSigWallet.transactions.history$)).find((t) => tx.id === t.id)!
+    );
+    // These are required because txBuilder still uses StakeRegistration
```

Comment:
We can flip the `isConwayEra` switch, or drop it. All networks are in conway era now, and we can use the newer certificates in TxBuilder too.
This can be another task if you want 😄

Reply:
Let's do it separately, and a little bit later to give people some time to update HW wallet apps

Reply:
Yep, the intention behind the complexity I added here is:
IDK when will be the right moment to consider the Conway era stable, I'll write a test in a wat that once the PR to remove pre-Conway stuff will be done:

1. this test will fail
2. the action to fix the test is the same action to reduce it to a single line

Please try to tolerate my obsessive compulsive disorders. 😜